### PR TITLE
Remove outdated information from README

### PR DESCRIPTION
**Pull request by laurenyu**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -32,29 +35,16 @@ Recommended
...
 
 The current master branch of this repository contains Dockerfiles and support code for MXNet versions 1.4.0 and higher.
 For MXNet version 1.3.0, check out v2.0.0 of this repository.
 For MXNet versions 0.12.1-1.2.1, check out v1.0.0 of this repository.
 
 For each supported MXNet version, Dockerfiles can be found for each processor type (i.e. CPU and GPU).
-For MXNet versions 0.12.1-1.3.0, the Dockerfiles produced images compatible for both training and serving.
-For MXNet versions 0.12.1 and 1.0.0, there are separate Dockerfiles for each Python version as well.
-
-All images are tagged with <mxnet_version>-<processor>-<python_version> (e.g. 1.3.0-cpu-py3).
```
Comment (ChoiByungWook) [inline README.rst]: I think you can keep this line

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://sbpy3lu9z2.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=bfa19579-0fc6-4c5b-be33-3bc2f1084ff3%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: README.rst [hunk 2]
```diff
@@ -32,29 +35,16 @@ Recommended
 Building Images
 ---------------
 
-The Dockerfiles in this repository are intended to be used for building Docker images to run training jobs and inference endpoints on `Amazon SageMaker <https://aws.amazon.com/documentation/sagemaker/>`__.
+The Dockerfiles in this repository are intended to be used for building Docker images to run training jobs on `Amazon SageMaker <https://aws.amazon.com/documentation/sagemaker/>`__.
 
 The current master branch of this repository contains Dockerfiles and support code for MXNet versions 1.4.0 and higher.
 For MXNet version 1.3.0, check out v2.0.0 of this repository.
```
[the remark this answers is not in the document]
Comment (laurenyu) [reply README.rst]: done

File: README.rst (hunk 1 above)
Comment (laurenyu) [reply README.rst]: it's been moved further down

File: README.rst [hunk 3]
```diff
@@ -32,29 +35,16 @@ Recommended
...
 
-Before building these images, you need to have two files already saved locally.
-The first is a pip-installable binary of the MXNet library.
-This can be something you compile from source or `download from PyPI <https://pypi.org/project/mxnet/#files>`__.
-
-The second is a pip-installable binary of this repository.
+Before building these images, you need to have a pip-installable binary of this repository saved locally.
 To create the SageMaker MXNet Container Python package:
 
 ::
```
[the remark this answers is not in the document]
Comment (laurenyu) [reply README.rst]: done

File: README.rst [hunk 4]
```diff
@@ -69,28 +59,26 @@ To create the SageMaker MXNet Container Python package:
 
 Once you have those binaries, you can then build the image.
 
-The Dockerfiles expect two build arguments:
+The Dockerfiles expect one build argument: ``py_version``.
+This is the Python version (i.e. 2 or 3).
 
-- ``py_version``: the Python version (2 or 3)
-- ``framework_installable``: the path to the MXNet binary
+The integration tests expect the Docker images to be tagged as ``preprod-mxnet:<tag>``, where ``<tag`` looks like <mxnet_version>-<processor>-<python_version> (e.g. 1.3.0-cpu-py3).
```
[the remark this answers is not in the document]
Comment (laurenyu) [reply README.rst]: done

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://sbpy3lu9z2.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=a8986bde-65a2-4bba-8e07-d55b944388b9%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**